manifests/k/Kiwix/WikiMed/1.9.5.0/Kiwix.WikiMed.installer.yaml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 # Created using wingetcreate
 
 PackageIdentifier: Kiwix.WikiMed
```

Comment:
InstallerUrl is not reachable, I think the manifest validation step is likely attempting to download the installer without success and reporting manifest-installer-validation-error label

Reply:
Will review